src/webapp/central/central.js

```diff
@@ -22,7 +23,18 @@ export default function Central(
       });
 
   const handler = callback => state.characteristic.startNotifications().then(() => {
-    const listerner = event => callback(decode(event.target.value));
+    const listerner = (event) => {
+      const chunk = decode(event.target.value);
+      const message = `${state.message}${chunk}`;
+
+      if (message.startsWith('[[[') && message.endsWith(']]]')) {
+        const json = JSON.parse(message.slice(3, message.length - 3));
```

Comment:
I would suggest to add a try/catch surrounding `JSON.parse` since it will throw an exception if the message is malformed 👍

Reply:
Issue created: https://github.com/jvallelunga/redux-bluetooth/issues/13